### PR TITLE
Add ID translation for Scale You App tutorial

### PR DESCRIPTION
**Pull request by supirman**
This PR is part of #22296 , translating `/docs/tutorials/kubernetes-basics/scale/` into Bahasa Indonesia. Also, update links and katakoda embed for other id k8s basic tutorials.

Notes:
 - instance : instans ([RSNI3](https://github.com/jk8s/sig-docs-id-localization-how-tos/blob/master/resources/RSNI-glossarium.pdf))
 - "replicas" are not translated since it is kubectl paramenter / Deployment field
 - scale out : perluasan skala
 - multiple : multipel https://kbbi.kemdikbud.go.id/entri/multipel
 - traffic : kunjungan https://twitter.com/ivanlanin/status/19299826772480000

/language id

### CODE REVIEW

**Review by habibrosyad — COMMENTED** [review]
Thank for your contribution @supirman! I just have a few suggestions regarding this PR...

File: content/id/docs/tutorials/kubernetes-basics/expose/expose-interactive.html [hunk 1]
```diff
@@ -21,12 +21,12 @@
             <div class="katacoda__alert">
                 Untuk berinteraksi dengan Terminal, harap gunakan desktop/tablet
             </div>
-            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/8" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-font="Roboto" data-katacoda-fontheader="Roboto Slab" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
+            <div class="katacoda__box" id="inline-terminal-1" data-katacoda-lang="id" data-katacoda-id="kubernetes-bootcamp/8" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-font="Roboto" data-katacoda-fontheader="Roboto Slab" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
```
Comment (habibrosyad) [inline content/id/docs/tutorials/kubernetes-basics/expose/expose-interactive.html:24]:
```suggestion
            <div class="katacoda__box" id="inline-terminal-1" data-katacoda-lang="id" data-katacoda-id="kubernetes-bootcamp/8" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
```
There's an update in https://github.com/kubernetes/website/pull/23762

File: content/id/docs/tutorials/kubernetes-basics/scale/scale-interactive.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+<div class="layout" id="top">
+
+    <main class="content katacoda-content">
+
+        <div class="katacoda">
+            <div class="katacoda__alert">
+                Untuk berinteraksi dengan Terminal, harap gunakan desktop/tablet
+            </div>
+            <div class="katacoda__box" id="inline-terminal-1" data-katacoda-lang="id" data-katacoda-id="kubernetes-bootcamp/5" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-font="Roboto" data-katacoda-fontheader="Roboto Slab" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
```
Comment (habibrosyad) [inline content/id/docs/tutorials/kubernetes-basics/scale/scale-interactive.html:24]:
```suggestion
            <div class="katacoda__box" id="inline-terminal-1" data-katacoda-lang="id" data-katacoda-id="kubernetes-bootcamp/5" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
```
Ditto, I think you haven't pulled from the latest master yet.

File: content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+          <h3>Tujuan</h3>
+                <ul>
+                    <li>Penyekalaan aplikasi menggunakan kubectl.</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+       <h3>Penyekalaan sebuah Aplikasi</h3>
+
+            <p>Di modul-modul sebelumnya kita telah membuat <a href="/id/docs/concepts/workloads/controllers/deployment/">Deployment</a>, dan mengeksposnya secara publik via <a href="/id/docs/concepts/services-networking/service/">Service</a>. Deployment yang dibuat itu hanya satu Pod yang menjalankan aplikasi kita. Ketika kunjungan meningkat, kita perlu menyekalakan (<i>scale</i>) aplikasi kita untuk mengikuti tingkat permintaan pengguna.</p>
```
Comment (habibrosyad) [inline content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html:30]:
```suggestion
            <p>Di modul-modul sebelumnya kita telah membuat <a href="/id/docs/concepts/workloads/controllers/deployment/">Deployment</a>, dan mengeksposnya secara publik via <a href="/id/docs/concepts/services-networking/service/">Service</a>. Deployment tersebut hanya membuat satu Pod yang menjalankan aplikasi kita. Ketika kunjungan meningkat, kita perlu melakukan penyekalaan (<i>scale</i>) aplikasi untuk mengikuti tingkat permintaan pengguna.</p>
```

File: content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+            <p><b>Penyekalaan</b> dapat dicapai dengan mengubah nilai replicas dalam Deployment</p>
```
Comment (habibrosyad) [inline content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html:32]:
```suggestion
            <p><b>Penyekalaan</b> dapat dicapai dengan mengubah jumlah replika dalam Deployment</p>
```

File: content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_lined">
+                    <h3>Ringkasan:</h3>
+                    <ul>
+                        <li>Penyekalaan sebuah Deployment</li>
+                    </ul>
+                </div>
+                <div class="content__box content__box_fill">
+                    <p><i> Kamu dapat dari awal memulai Deployment dengan multipel instans menggunakan parameter --replicas pada perintah kubectl create deployment</i></p>
```
Comment (habibrosyad) [inline content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html:43]:
```suggestion
                    <p><i> Kamu dapat memulai Deployment dengan beberapa instans sekaligus dari awal dengan menggunakan parameter --replicas pada perintah kubectl create deployment</i></p>
```

File: content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+        </div>
+
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+
+                <p>Perluasan skala Deployment akan memastikan Pod baru dibuat dan dijadwalkan ke Node-Node dengan sumber daya yang tersedia. Penyekalaan akan meningkatkan jumlah Pod ke keadaan yang diinginkan. Kubernetes juga mendukung <a href="/id/docs/tasks/run-application/horizontal-pod-autoscale/"><i>autoscaling</i></a> Pod, tetapi itu di luar cakupan tutorial ini. Penyekalaan ke nol juga dimungkinkan, dan tindakan ini akan mengakhiri semua Pod pada Deployment tersebut.</p>
+
+                <p>Menjalankan beberapa instans dari aplikasi akan membutuhkan cara untuk mendistribusikan kunjungan ke semuanya. Service punya penyeimbang beban terintegrasi yang akan mendistribusikan kujungan jaringan ke semua Pod dari sebuah Deployment yang diekspos. Service akan terus-menerus memonitor Pod-Pod yang berjalan menggunakan Endpoints, untuk memastikan kunjungan hanya dikirim kepada Pod yang tersedia.</p>
```
Comment (habibrosyad) [inline content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html:91]:
```suggestion
                <p>Menjalankan beberapa instans dari aplikasi akan membutuhkan cara untuk mendistribusikan trafik ke semuanya. Service memiliki penyeimbang beban terintegrasi yang akan mendistribusikan trafik jaringan ke semua Pod dari sebuah Deployment yang diekspos. Service akan terus memonitor Pod-Pod yang berjalan menggunakan Endpoints, untuk memastikan trafik hanya dikirim ke Pod yang tersedia.</p>
```
Ref for `trafik`: [this sheets](https://docs.google.com/spreadsheets/d/1uK1TGfKyT5ZmWWQx4c4KYm7YmOmuOHyQW-bh8rGTMP4/edit#gid=0)

File: content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_fill">
+                    <p><i>Penyekalaan dapat dicapai dengan mengubah jumlah replicas pada sebuah Deployment.</i></p>
```
Comment (habibrosyad) [inline content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html:96]:
```suggestion
                    <p><i>Penyekalaan dapat dicapai dengan mengubah jumlah replika pada sebuah Deployment.</i></p>
```

File: content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+                </div>
+            </div>
+        </div>
+
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p>Sewaktu kamu punya multipel instans dari aplikasi berjalan, kamu akan dapat melakukan pembaruan bertahap tanpa perhentian. Kita akan akan membahas hal tersebut pada modul selanjutnya. Sekarang, mari kita pergi ke terminal daring dan menyekalakan aplikasi kita.</p>
```
Comment (habibrosyad) [inline content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html:105]:
```suggestion
                <p>Sewaktu kamu memiliki multipel instans dari aplikasi yang berjalan, kamu akan bisa melakukan pembaruan bertahap tanpa perhentian. Kita akan akan membahas hal tersebut pada modul selanjutnya. Sekarang, mari kita pergi ke terminal daring dan melakukan penyekalaan terhadap aplikasi kita.</p>
```

File: content/id/docs/tutorials/kubernetes-basics/scale/scale-interactive.html (hunk 2 above)
Comment (supirman) [reply content/id/docs/tutorials/kubernetes-basics/scale/scale-interactive.html:24]: Yeah, I forget to update my master before making the copy from the original article. Thanks.

File: content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html (hunk 4 above)
Comment (supirman) [reply content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html:32]: this one arguable because of this reverence to `.spec.replicas` field on the Deployment.

Comment (habibrosyad) [reply content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html:32]: Yes, I know that there's a `replicas` field for Deployment `spec`. When I read the original text I think it more as a general sentence, since it's not specifically mentioning that replicas is a field there (it's ambiguous whether it means replicas in plain English tech term or `replicas` field). Taken for granted updating the `replicas` field value means changing the number of replicas for a Deployment.

It's just my personal view btw...

Comment (habibrosyad) [reply content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html:32]: More discussions here https://kubernetes.slack.com/archives/CJ1LUCUHM/p1600235247012200

File: content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html (hunk 3 above)
Comment (ariscahyadi) [inline content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html:30]: nit! just to be clearer!
```suggestion
            <p>Di modul-modul sebelumnya kita telah membuat <a href="/id/docs/concepts/workloads/controllers/deployment/">Deployment</a>, dan mengeksposnya secara publik via <a href="/id/docs/concepts/services-networking/service/">Service</a>. Deployment yang dibuat hanya mempunyai satu Pod untuk menjalankan aplikasi kita. Ketika kunjungan meningkat, kita perlu menyekalakan (<i>scale</i>) aplikasi kita untuk mengikuti tingkat permintaan pengguna.</p>
```

File: content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html (hunk 8 above)
Comment (ariscahyadi) [inline content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html:105]: nit!
```suggestion
                <p>Sewaktu kamu punya multipel instans dari aplikasi berjalan, kamu akan dapat melakukan pembaruan bertahap tanpa henti. Kita akan akan membahas hal tersebut pada modul selanjutnya. Sekarang, mari kita pergi ke terminal daring dan menyekalakan aplikasi kita.</p>
```

File: content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-12">
+                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/scale/scale-interactive/" role="button"> Mulai Tutorial Interaktif <span class="btn__next">›</span></a>
```
Comment (ariscahyadi) [inline content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html:112]: Link to ID doc, if available.
```suggestion
                <a class="btn btn-lg btn-success" href="/id/docs/tutorials/kubernetes-basics/scale/scale-interactive/" role="button"> Mulai Tutorial Interaktif <span class="btn__next">›</span></a>
```

**Review by ariscahyadi — COMMENTED** [review]
Thanks, @supirman for your contributions and @habibrosyad for your comments.
There are still a few comments, but no need to be addressed right now.

/lgtm

File: content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html (hunk 5 above)
Comment (supirman) [reply content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html:43]:
```suggestion
                    <p><i> Kamu dapat membuat Deployment dengan beberapa instans sekaligus dari awal dengan menggunakan parameter --replicas pada perintah kubectl create deployment</i></p>
```

File: content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html (hunk 3 above)
Comment (supirman) [reply content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html:30]:
```suggestion
            <p>Di modul-modul sebelumnya kita telah membuat <a href="/id/docs/concepts/workloads/controllers/deployment/">Deployment</a>, dan mengeksposnya secara publik via <a href="/id/docs/concepts/services-networking/service/">Service</a>. Deployment tersebut hanya membuat satu Pod untuk menjalankan aplikasi kita. Ketika kunjungan meningkat, kita perlu menyekalakan (<i>scale</i>) aplikasi kita untuk mengikuti tingkat permintaan pengguna.</p>
```

File: content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html (hunk 8 above)
Comment (supirman) [reply content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html:105]:
```suggestion
                <p>Sewaktu kamu memiliki multipel instans dari aplikasi yang berjalan, kamu akan bisa melakukan pembaruan bertahap tanpa henti. Kita akan akan membahas hal tersebut pada modul selanjutnya. Sekarang, mari kita pergi ke terminal daring dan melakukan penyekalaan terhadap aplikasi kita.</p>
```

**Review by ariscahyadi — COMMENTED** [review]
It is good to go!

/lgtm

**Comment by irvifa** [comment]
Thanks for the awesome job
/approve
